### PR TITLE
Updated RC common and dummy_connecttest

### PR DESCRIPTION
**Pull request by aderiabin**

### CODE REVIEW

File: user_modules/dummy_connecttest.lua [hunk 1]
```diff
@@ -464,4 +466,4 @@ function module:startSession()
   return mobile_connected
 end
 
-return module
+return module
```
Comment (dboltovskyi) [inline user_modules/dummy_connecttest.lua]: @aderiabin Please add new line

File: test_scripts/RC/commonRC.lua [hunk 2]
```diff
@@ -635,4 +637,41 @@ function commonRC.rpcRejectWithoutConsent(pModuleType, pAppId, pRPC, self)
...
+      capParams.buttonCapabilities = pButtonCapabilities
+    end
+  else
+    capParams.buttonCapabilities = nil
+  end
+
+  return hmiParams
+end
+
+return commonRC
```
Comment (dboltovskyi) [inline test_scripts/RC/commonRC.lua]: @aderiabin Please add new line

File: test_sets/rc_GetCapabilities.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+./test_scripts/RC/GetCapabilities/001.lua
```
Comment (aderiabin) [inline test_sets/rc_GetCapabilities.txt]: @dboltovskyi Fixed in 29308ba

File: user_modules/dummy_connecttest.lua (hunk 1 above)
Comment (aderiabin) [inline user_modules/dummy_connecttest.lua]: @dboltovskyi Fixed in 29308ba

File: test_scripts/RC/commonRC.lua (hunk 2 above)
Comment (aderiabin) [inline test_scripts/RC/commonRC.lua]: @dboltovskyi Fixed in 29308ba

**Pull request merged**